public/apps/configuration/utils/password-edit-panel.tsx

```diff
@@ -43,7 +62,7 @@ export function PasswordEditPanel(props: {
 
   return (
     <>
-      <FormRow headerText="Password" helpText={PASSWORD_INSTRUCTION}>
+      <FormRow data-test-subj="password-form-row" headerText="Password" helpText={passwordHelpText}>
```

Comment:
qq. Why add `data-test-subj`?

Reply:
I removed that, I was trying to write a test to verify the text after a state update in the password-edit-panel but opted instead to assert that setState with the custom message had been called.